src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
if you want the facades to come into play when this is used in netframework, we should cross target this for netfx.

cc: @joperezr

Reply:
That is correct. That said, I still have to merge all of the netstandard facades work into dotnet/runtime, so I don't mind taking care of this one as well once I do that so as to not introduce m ore changes into this PR.